uqcsbot/api.py

```diff
@@ -226,7 +229,6 @@ def _initialise(self):
                     if im['is_user_deleted']:
                         continue
 
-                    im['name'] = im['id']
                     im['is_private'] = True
```

Comment:
why was name removed here? is name available in im.list? or were you meant to remove is_private? (as you did in the lower change)

Reply:
Yeah that was a mistake, thanks